.gitignore

```diff
@@ -4,3 +4,5 @@ _site
 .DS_Store
 .idea
 docs/build
+node_modules
+package-lock.json
```

Comment:
This is what I put into my `.git/info/exclude`:

```
docs/node_modules/
docs/build/
docs/package-lock.json
```

Reply:
I mean -- we already ignore `docs/build`, so why not `docs/node_modules` and `docs/package-lock.json`?

Reply:
Someone might install the node modules into the root as well, the `node_modules` and `package-lock.json` things don't necessarily have to be under `docs`.

Reply:
Hmm, I guess that's because you can `npm install` Antora in the root directory and just run `npx antora generate docs/antora-playbook.yml`? Alright, sorry for those silly questions :-)